### ISSUE TITLE
Okta browser login page does not appear on Android

### ISSUE THREAD

**Issue by CodingItWrong** [okta/samples-js-react-native#30]
I'm attempting to get `browser-sign-in` working on Android. I've followed the instructions in the README to create a personal Okta account and application, and configure it just as the README specifies.

When I attempt to run the application, on all device configurations I tested, I see the Login button, but tapping it does not bring up the Okta browser login page. Adding some logging statements, the call to `@okta/okta-react-native`'s `signIn()` function returns a promise that resolves quickly (no arguments passed to the resolve handler).

This behavior happens on the following configurations I tested:
- Emulator: Pixel C API 28 (Android 9.0, x86)
- Emulator: Pixel 3 API 28 (Android 9.0, x86)
- Hardware device: Galaxy Tab A SM-T510, Android 9

**Comment by kuldip9060** [comment]
Same thing is happening with my project when I click on login button nothing is happening since last 2-3 days.

**Comment by CodingItWrong** [comment]
The branch `browser-sign-in-redirect-schema-doc` corresponds to #31, and I've confirmed following those instructions gets the login page appearing for me. I opened #32 as an alternate solution.

Once one of those two PRs is merged we can close this issue.

**Comment by swiftone** [comment]
@CodingItWrong - Thanks for the suggestion.  We're rolling with #31 (which points to docs describing the change required).  

We'll update here once that change is live, though as this is a documentation change it looks like no one is blocked.  Let us know if that's not accurate.

**Comment by shuowu-okta** [comment]
#31 has been merged into master. Close this issue.

**Comment by bearcub3** [comment]
I do have the same issue with the the most recent version of react native and the okta auth dependecy. has it been fixed? any update regarding this?

**Comment by mraible** [comment]
I tested `"@okta/okta-react-native": "1.4.4"` with an Android emulator last week and was able to make it work. Are you testing on an actual device or an emulator?

**Comment by bearcub3** [comment]
I've tested on a Pixel 3 emulator.  `"@okta/okta-react-native": "^1.4.4", "react-native": "0.63.4",`  
I'd like to get the authorization code for the custom authorization server(access management). so far, I only have been successful getting to work on the custom-sign-in which doesn't seem to be able to get the code. any advice or tips please?

**Comment by blairnangle** [comment]
I am experiencing the same issue but only on physical Android devices (emulator and iOS (physcial device and simulator) are fine). I am using:

```json
"react-native": "0.63.4",
@okta/okta-react-native": "^1.4.4",
```

I am using a custom sign in component.

**Comment by dewa1993** [comment]
On a physical device I solved it by switching default browser to chrome. In my case the default browser was set to Samsung's default browser. 
Tried this with couple of devices and browsers other than chrome and faced the same issue.
1) Samsung A51
2) Redmi Note 9
